plugins/services/src/js/components/dsl/ServiceHealthDSLSection.js

```diff
@@ -0,0 +1,86 @@
+import React, {PropTypes} from 'react';
+
+import DSLCombinerTypes from '../../../../../../src/js/constants/DSLCombinerTypes';
+import DSLExpression from '../../../../../../src/js/structs/DSLExpression';
+import DSLExpressionPart from '../../../../../../src/js/structs/DSLExpressionPart';
+import DSLFormWithExpressionUpdates from '../../../../../../src/js/components/DSLFormWithExpressionUpdates';
+import DSLUtil from '../../../../../../src/js/utils/DSLUtil';
+import FieldInput from '../../../../../../src/js/components/form/FieldInput';
+import FieldLabel from '../../../../../../src/js/components/form/FieldLabel';
+import FormGroup from '../../../../../../src/js/components/form/FormGroup';
+
+const PARTS = {
```

Comment:
The `PARTS` refers to DSL Expression parts, but I am open to suggestions :|
So, perhaps `EXPRESSION_PARTS` ?